src/module/MintFeeManagerModule.sol

```diff
@@ -0,0 +1,153 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.20;
+
+import {Module} from "../Module.sol";
+import {Role} from "../Role.sol";
+
+library MintFeeManagerStorage {
+
+    /// @custom:storage-location erc7201:mint.fee.manager
+    bytes32 public constant MINT_FEE_MANAGER_STORAGE_POSITION =
+        keccak256(abi.encode(uint256(keccak256("mint.fee.manager")) - 1)) & ~bytes32(uint256(0xff));
+
+    struct Data {
+        address platformFeeRecipient;
```

Comment:
feeRecipient is good enough since platformFee means something different in the context of some of our contracts

Reply:
maybe we can combine platformFee with our fee

Reply:
not too sure what you mean by combining platformFee with our Fee.